ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -537,16 +572,16 @@ mapValuesAppliedMK f = \case
   ApplyQueryAllMK         -> ApplyQueryAllMK
   ApplyQuerySomeMK ks     -> ApplyQuerySomeMK (castUtxoKeys ks)
 
-diffKeysMK :: ApplyMapKind DiffMK k v -> ApplyMapKind KeysMK k v
+diffKeysMK :: DiffMK k v -> KeysMK k v
```

Comment:
Here we use the `DiffMK` type alias, whereas above we use `ApplyMapKind' KeysMK'` for instance. Would it make sense to use either one or the other?

Reply:
I accidentally removed most `ApplyMapKind` within this module. Before this PR is merged, subsequent commits that only rename things or inline type synonyms etc (entirely mechanical) should address these things. We ultimately want to remove `ApplyMapKind`, leaving `DiffMK` etc as the only "public" interface.